decidim-elections/lib/decidim/votings/admin_engine.rb

```diff
@@ -30,10 +30,12 @@ class AdminEngine < ::Rails::Engine
 
           resource :census, only: [:show, :destroy], controller: "/decidim/votings/census/admin/census" do
             member do
-              get :status, action: :status
-              post :create, action: :create
-              put :update, action: :update
-              get :generate_access_codes, action: :generate_access_codes
+              get :status
+              post :create
+              put :update
```

Comment:
Could we move these two to line 31?

Reply:
I'll also remove the `update` action, since it is not required anymore.

Reply:
Done in https://github.com/decidim/decidim/pull/7756/commits/37c37ff6032c28a6700f76501d7da6f18c2acdf3